sw/device/lib/dif/dif_hmac_unittest.cc

```diff
@@ -114,4 +114,49 @@ TEST_F(HmacMacTest, StartError) {
             kDifError);
 }
 
+class HmacSha256Test : public HmacTest {
+ protected:
+  HmacSha256Test() { config_reg_.sha_enable = true; }
+
+  void SuccessPath() {
```

Comment:
This is only used three times. I'd inline it. See https://stackoverflow.com/questions/6453235/what-does-damp-not-dry-mean-when-talking-about-unit-tests for why this kind of DRY hinders rather than helps.

Reply:
Done